syntax = "proto3";

package bakdata.stock.v1;

message StockCorporate {
  string id = 1;
  uint32 stock_id = 2;
  string symbol = 3;
  string company_name = 4;
  string last_refreshed = 5;
  string time_zone = 6;
  repeated StockEntry stockEntry = 7;
}

message StockEntry {
  string id = 1;
  string date = 2;
  string open = 3;
  string close = 4;
  string high = 5;
  string low = 6;
  string volume = 7;
}